python/sdk/tests/test_scm.py

```diff
@@ -42,22 +37,36 @@ def test_get_policy_evaluation_summary(self, unwrap):
             if controls := categories[0].get('controls'):
                 assert {'control', 'inbox_count', 'setting_count', 'setting_misconfiguration_count'} == controls[0].keys()
 
-@pytest.mark.order(10)
-@pytest.mark.usefixtures('setup_account')
-class TestPolicyEvaluation:
-    controls = [
-        ('crowdstrike', True),
-        ('defender', True),
-        ('sentinelone', True),
-        ('intune', False),
-    ]
+    def test_technique_summary(self, unwrap):
+        summary = unwrap(self.scm.technique_summary)(self.scm, 'T1078,T1027')
+        assert len(summary) > 0
+        assert {'controls', 'technique'} == summary[0].keys()
+        assert len(summary[0]['controls']) > 0
+        assert {'control', 'setting_count', 'setting_misconfiguration_count'} == summary[0]['controls'][0].keys()
+
+    def test_export_missing_edr_endpoints_csv(self, unwrap):
+        if not pytest.expected_account['features']['policy_evaluator']:
+            pytest.skip('POLICY_EVALUATOR feature not enabled')
+        csv = unwrap(self.export.export_scm)(self.export, 'endpoints/?$filter=missing_edr eq true&$top=2').decode('utf-8')
+        assert len(csv.split('\n')) == 3
```

Comment:
sometimes it returns an empty line at the end so would be better to add a strip, i.e. `len(csv.strip('\r\n').split('\r\n)) == 3`